lizmap/modules/lizmap/classes/lizmapOGCRequest.class.php

```diff
@@ -106,6 +165,31 @@ protected function buildQuery($params)
         return str_replace($a, $b, $bparams);
     }
 
+    protected function request($post=False)
+    {
+        $querystring = $this->constructUrl();
+
+        $options = array();
+        if ($this->requestXml !== Null) {
+            $options = array(
+                'method' => 'post',
+                'headers' => array('Content-Type' => 'text/xml'),
+                'body' => $this->xmlRequest,
```

Comment:
`$this->requestXml` instead of `$this->xmlRequest` ?

Reply:
Thanks, it's a bug